### ISSUE TITLE
Sublime Text 3 support

### ISSUE THREAD

**Issue by toph-allen** [joelpt/sublimetext-print-to-html#10]
Any plans to support ST3 in the near future? I'm not familiar with the API changes; how much work would that take above converting to Python 3?

**Comment by glukki** [comment]
+1

**Comment by alessandro-molari** [comment]
:+1: :+1: :+1: 

In ST3 nothing works:
- _Print as HTML..._ is grayed out
- Keybinding `alt+shift+p` doesn't work
- `ctrl+p` `print` doesn't work.

**Comment by msheaver** [comment]
I agree, it does not work for me in ST3; I get the same results. Are there any plans to port this to ST3?

**Comment by MattHulse** [comment]
I would like to help get this working in ST3.  Do you know what the roadblocks are?  Seems like Pygments has Python 3 support.

**Comment by MattHulse** [comment]
While it's not official, I got Print To HTML working in ST3 for my own use.  My fork is https://github.com/MattHulse/sublimetext-print-to-html

Mostly the work involved running 2to3 on the pygments and PrintToHTML.py code.  In addition I had to add a file to the repo so a working directory was still extracted instead of just a zipped package file when using package manager.  I also had to modify the load path and the loading of resources in the working directory due to changes in ST3.

It's working well for me.  YMMV

**Comment by toph-allen** [comment]
It's working for me too. Thanks for putting the time in to get it running!

**Comment by MattHulse** [comment]
Updated my fork to fix 'print to new buffer' command and bug that occurred on xfce desktop.

**Comment by KyleKing** [comment]
@MattHulse, I looked for your fork, but I couldn't find it via package control. Is it only available through manual install?

**Comment by MattHulse** [comment]
Yeah, it would have only been available via manual install.  However, I'm not sure it would even work after this long.  I haven't used it for at least a year.

**Comment by ralyodio** [comment]
how do i install this damn thing.
